src/main/java/com/xceptance/xlt/report/providers/ConfigurationReportProvider.java

```diff
@@ -138,6 +142,46 @@ public Object createReportFragment()
         return report;
     }
 
+    private void calculatePercentages(List<TestCaseLoadProfileConfiguration> loadProfile)
+    {
+        // iterate through report.loadProfile --> calculate totals of (max) users and (max) arrivalrate
+        int arrivalRateTotal = 0;
+        int userCountTotal = 0;
+        for (TestCaseLoadProfileConfiguration loadProfileConfig : loadProfile)
+        {
+            int[][] arrivalRateFunction = loadProfileConfig.getArrivalRate();
+            arrivalRateTotal += getMaxFromLoadFunction(arrivalRateFunction);
+            int[][] userCountFunction = loadProfileConfig.getNumberOfUsers();
+            userCountTotal += getMaxFromLoadFunction(userCountFunction);
+            
+        }
+        // iterate through report.loadProfile again and add percentages calculated from totals
+        for (TestCaseLoadProfileConfiguration loadProfileConfig : loadProfile)
+        {
+            int[][] arrivalRateFunction = loadProfileConfig.getArrivalRate();
+            loadProfileConfig.setArrivalRatePercentage(ReportUtils.calculatePercentage(getMaxFromLoadFunction(arrivalRateFunction), arrivalRateTotal));
+            int[][] userCountFunction = loadProfileConfig.getNumberOfUsers();
+            loadProfileConfig.setNumberOfUsersPercentage(ReportUtils.calculatePercentage(getMaxFromLoadFunction(userCountFunction), userCountTotal));
+            
+        }        
+        return;
+    }
+    
+    private int getMaxFromLoadFunction(int[][] loadFunction)
```

Comment:
If an arrival rate is not defined for a scenario, the load function is null, so this method will produce a NPE.